src/main/java/seedu/address/logic/commands/addcommand/AddMentorCommand.java

```diff
@@ -15,7 +15,7 @@
  */
 public class AddMentorCommand extends AddCommand {
 
-    public static final String COMMAND_WORD = "addMentor";
+    public static final String COMMAND_WORD = "mentor";
```

Comment:
Instead of leaving this as `COMMAND_WORD`, in `CliSyntax` we have `ENTITY_MENTOR` etc

Reply:
Yea good spot actually. But i realised that in the MESSAGE_USAGE there is reference to COMMAND_WORD several times when explaining the format, which is why I left it as such. Is that fine, or should I maybe rename it/change it?